src/keycloak/index.js

```diff
@@ -5,7 +5,8 @@ let keycloak = Keycloak();
 let initOptions = {
     responseMode: 'fragment',
     flow: 'standard',
-    onLoad: 'login-required'
+    onLoad: 'login-required',
+    pkceMethod: 'S256'

```

Comment:
I don't understand why GitHub is reporting this change. It's already on the master branch. I guess it must be reporting the diff based on where you branched? Not a big deal, just surprising and I don't understand it.